lib/deployed_version.rb

```diff
@@ -40,20 +40,20 @@ def minor
       @minor ||= version(:minor)
     end
 
+    def patch
+      @patch ||= version(:patch)
+    end
+
     def extra
       @extra ||= version(:extra)
     end
 
     def version_hash
-      @version_hash ||= /\Arelease-(?<major>\d+)\.(?<minor>\d+)\.?(?<extra>\S*)\z/.match(label)
+      @version_hash ||= /\Av(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)(?<extra>\S*)/.match(label)
     end
 
     def version_label
-      if version_hash
-        "#{major}.#{minor}.#{extra}"
-      else
-        'WIP'
-      end
+      major == 0 && minor == 0 && patch == 0 ? 'WIP' : "#{major}.#{minor}.#{patch}"
```

Comment:
I know you pulled this in from the other repos, but I think the version here (minus the change to 'patch') was actually correct.

If a rexep fails to match, it will return nil, which means the 

```ruby
    def version(rank)
      version_hash ? version_hash[rank] : '0' # String, as it matches what we'd get from the regex
    end
```
Will always return the string '0'. Yet the check here looks for equality with the number.

Reply:
Thanks!  Good point.  I hadn't appreciate there was a difference here.